modules/zeromq-ext/pom.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+<!--
+    POM file.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.apache.ignite</groupId>
+        <artifactId>ignite-parent</artifactId>
+        <version>1</version>
+        <relativePath>../../parent</relativePath>
+    </parent>
+
+    <artifactId>ignite-zeromq-ext</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <url>http://ignite.apache.org</url>
+
+    <properties>
+        <jeromq.version>0.3.6</jeromq.version>
+        <ignite-core.version>2.9.0-SNAPSHOT</ignite-core.version>
+        <ignite-log4j.version>2.9.0-SNAPSHOT</ignite-log4j.version>
+        <ignite-spring.version>2.9.0-SNAPSHOT</ignite-spring.version>
```

Comment:
it should be a single ignite.version, we do not need to have a separate version for each jar.
Furthermore, why do we have it here? I think it should be defined in parent pom, so all extensions will be built and release for the same ignite version. However, if we want to treat all extensions as absolutely independent unites and each one can support different ignite versions, then why we put all extensions under a single umbrella project?

Reply:
@Mikhail1988 I have updated the PR